core/src/main/resources/error/error-classes.json

```diff
@@ -2117,6 +2117,11 @@
           "Table <tableName> does not support <operation>. Please check the current catalog and namespace to make sure the qualified table name is expected, and also check the catalog implementation which is configured by \"spark.sql.catalog\"."
         ]
       },
+      "TIME_TRAVEL" : {
+        "message" : [
+          "Time travel on: <target>"
```

Comment:
Wouldn't be more helpful for users to show some id of the target (table name, for instance) instead of the generic string `path-based tables`? This could help in troubleshooting the issue.

Reply:
Very good proposal, let me do it.